typescript/infra/src/agents/key-utils.ts

```diff
@@ -438,26 +443,25 @@ export async function persistValidatorAddressesToLocalArtifacts(
       validators: fetchedValidatorAddresses[chain].validators, // fresh from aws
     };
   }
-  // Resolve the relative path
-  const filePath = path.resolve(__dirname, '../../config/aw-multisig.json');
   // Write the updated object back to the file
-  fs.writeFileSync(filePath, JSON.stringify(awMultisigAddresses, null, 2));
+  const dir = path.join(__dirname, '../../config');
+  writeJSON(
+    dir,
+    'aw-multisig.json',
+    JSON.stringify(awMultisigAddresses, null, 2),
+  );
 }
 
-export function fetchLocalKeyAddresses(
-  role: Role,
-  environment: DeployEnvironment,
-  context: Contexts,
-): Address {
-  // Resolve the relative path
-  const filePath = path.resolve(__dirname, `../../config/${role}.json`);
-  const data = fs.readFileSync(filePath, 'utf8');
-  const addresses: LocalRoleAddresses = JSON.parse(data);
+export function fetchLocalKeyAddresses(role: Role): LocalRoleAddresses {
+  try {
+    const dir = path.join(__dirname, '../../config');
```

Comment:
can you pull this out into a global const?

Reply:
fixed